ppr-ui/vite.config.ts

```diff
@@ -59,6 +59,8 @@ export default defineConfig(() => {
       },
       globals: true,
       threads: true,
+      minThreads: 4,
```

Comment:
Just curious, is this the memory issue fix?

Reply:
This is a limiter on the threads (unit tests running in parallel), need to set min/max in order to allow threads but restrict them to a certain count. Sort of a memory fix, more of memory preservation.

Reply:
But sorry, yes, exactly. This is my attempt to fix those memory issues! haha 
As well as increasing the memory defaults

Reply:
Do you know what was the default threads count?

Reply:
Sorry, didn't see this before!
I believe the amount of threads depends on your system/processor etc. It might just try and spin up as many as your processor will allow.
That being said, i think performance dips after 8, from what i've read at least. 
If your curious what your system tries, look at your console and see how test suites are running in parallel without the thread cap.